package.json

```diff
@@ -40,7 +40,7 @@
     "merge-stream": "^1.0.0",
     "minimatch-all": "^1.0.2",
     "multipipe": "^0.3.0",
-    "plylog": "^0.3.0",
+    "plylog": "^0.4.0",
```

Comment:
oh right. v0.4 included work to share verbosity/level state across loggers. I had originally hoped we could get away with setting the state early and then having all loggers properly load that state on creation. But with module loading taking place before user input is read, that was unrealistic. In v0.4 changing the level now affects all loggers, instead of just yet-to-be-created loggers.